src/core_seaice/shared/mpas_seaice_berg_velocity_solver.F

```diff
@@ -1014,6 +1057,13 @@ subroutine berg_atm_forcing(domain)
          airVertDragCoef = 0.2_RKIND, &
          airHorizDragCoef = 0.00025_RKIND
 
+    ! halo exchange
+    call mpas_timer_start("Berg velocity solver halo")
+    call MPAS_dmpar_field_halo_exch(domain, 'uAirVelocity')
+    call MPAS_dmpar_field_halo_exch(domain, 'vAirVelocity')
+    call MPAS_dmpar_field_halo_exch(domain, 'airDensity')
```

Comment:
Why do you need to halo exchange airDensity??

Reply:
It's not needed - leftover from testing. I'll remove it.